assetsys.h

```diff
@@ -3266,124 +3385,6 @@ void *tdefl_write_image_to_png_file_in_memory(const void *pImage, int w, int h,
 
 #ifndef MINIZ_NO_ARCHIVE_APIS
 
-#ifdef MINIZ_NO_STDIO
-  #define MZ_FILE void *
-#else
-  #include <stdio.h>
-  #include <sys/stat.h>
-
-  #if ( defined(_MSC_VER) && _MSC_VER >= 1400 ) || defined(__MINGW64__)
-    static FILE *mz_fopen(const char *pFilename, const char *pMode)
```

Comment:
Overriding the `MZ_FOPEN` defines with `ASSETSYS_FOPEN`, and keeping `MINIZ_NO_STDIO` required removing this `MINIZ_NO_STDIO`, as it hard-defines the values without `#ifndef` blocks.